ngui/ui/src/components/forms/ConnectCloudAccountForm/FormElements/ConnectionFields.tsx

```diff
@@ -114,24 +41,38 @@ const NebiusInputs = () => (
 
 type ConnectionType = ObjectValues<typeof CONNECTION_TYPES>;
 
-const ConnectionInputs = ({ connectionType }: { connectionType: ConnectionType }) => {
+const renderAwsInputs = (connectionType: ConnectionType, authenticationType: AuthenticationType | null): JSX.Element => {
```

Comment:
There is no need to explicitly define the return type; TypeScript will infer it based on the returned value


```suggestion
const renderAwsInputs = (connectionType: ConnectionType, authenticationType: AuthenticationType | null) => {
```

Reply:
done